lib/usb.ex

```diff
@@ -6,7 +6,7 @@ defmodule Circuits.USB do
 
   defdelegate list_devices, to: Nif
 
-  defdelegate get_handle(id_vendor, id_product), to: Nif
+  defdelegate get_handle(id_vendor, id_product, interface_num \\ 0), to: Nif
```

Comment:
What does `interface_num \\ 0` do?

Reply:
Sets the default to 0 if not passed. Just shorthand for supporting `get_handle/2` and `get_handle/3` functions